mixitup/teamInfo.js

```diff
@@ -0,0 +1,247 @@
+var context = 
```

Comment:
I am using the teamInfo.js file because it makes sense with the way that Handlebars works. As far as I know, there isn't a direct way to pull the information directly from the Google sheet (even to make it CSV you have to download it - it doesn't speak directly to it). There is embedded JSON because I think it is easier to work with and look at than JSON directly - but that may just be a personal preference. I can try working with the JSON rather than beautify-ing it in a new file but the process doesn't take that much time (DL sheet as CSV, run the script, and beautify the JSON in a new file). Does this make sense?